src/hyperlink/_url.py

```diff
@@ -742,7 +757,7 @@ def parse_host(host):
 
 
 class URL(object):
-    """From blogs to billboards, URLs are so common, that it's easy to
+    r"""From blogs to billboards, URLs are so common, that it's easy to
```

Comment:
Made this a raw string because there are `\` characters in there which are not actually trying to escape the next character and chasing those to `\\` actually wrecks the string's mojo.

Reply:
as long as it looks fine in the docs :) We'll see what sphinx/readthedocs does with it.